django_sharding_library/decorators.py

```diff
@@ -45,8 +46,14 @@ def configure(cls):
                 try:
                     if not isinstance(cls.objects, ShardManager):
                         if type(cls.objects) == Manager:
-                            cls.add_to_class('objects', ShardManager())
-                            cls._base_manager = cls.objects
+                            to_add = ShardManager()
+                            to_add.name = 'shard_manager'
```

Comment:
I think this is the issue and that if I override the name on init of the inherited manager, it will work.